samples/v1/types_config.go

```diff
@@ -64,6 +64,30 @@ type ConfigSpec struct {
 	// content but the operator will not recreate(or update) anything
 	// listed here.
 	SkippedTemplates []string `json:"skippedTemplates,omitempty" protobuf:"bytes,6,opt,name=skippedTemplates"`
+
+	// helmChartList specifies the list of the helmcharts(S2i) that
+	// the samples operator will install
+	// +listType=map
+	// +listMapKey=name
+	HelmChartList []HelmChart `json:"helmChartList,omitempty" protobuf:"bytes,7,opt,name=helmChartList"`
+}
+
+type HelmChart struct {
+	// URL of the helmchart which is going to be installed
+	// +kubebuilder:validation:Pattern=`^https?:\/\/`
+	// +kubebuilder:validation:MaxLength=2048
+	URL string `json:"url,omitempty"`
+
+	//Namespace where the helmchart is to be installed (openshift)
```

Comment:
is this supposed to convey that the default NS is openshift?  If so, please state it as such.

Reply:
to make it easier and avoid confusion, i have updated the list of parameters and removed the namespace altogether, as it will be "openshift" only